examples/snapshot/basic_snap.py

```diff
@@ -0,0 +1,47 @@
+""" Sample script to demonstrate use of SoCo snapshot
+
+This is useful for scenarios such as when you want to switch to radio,
+an announcement or doorbell sound and then back to what was playing previously.
+
+To use it just change the IP address to one of your Sonos players. 
+
+This script will:
+ - start playing a radio station
+ - take a snapshot of the status of the sonos player (what playing, volume etc.)
+ - play an alert mp3 for a few seconds (a poem!)
+ - re-instate the sonos player to it's previous state (the radio station)
+ 
+Note: the snap function is designed to be used once, Having taken a snap to 
```

Comment:
3 things:

Capital after ":"

Calling if function might confuse since it isn't actually a function, so either change to functionality or say that the "Snapshot instance" is only designed to be used once.

"once, Having", change to full stop or make lowercase. Also the last part if that sentence is maybe a little to compactly formulated. What do you think?